src/helpers/configuration_helpers.sh

```diff
@@ -52,6 +69,22 @@ function get_configuration {
 }
 
 
+# Returns whether or not Git Town is configured
+function is_git_town_configured {
+  if [ -n "$main_branch_name" ] && get_configuration 'non-feature-branch-names'; then
+    echo true
+  else
+    echo false
+  fi
+}
+
+
+# Remove all Git Town configuration
+function remove_all_configuration {
+  git config --remove-section git-town > /dev/null 2>&1
```

Comment:
Other people will have the same question as the reviewer. Please add a comment that explains this motivation for redirecting the output here.